docs/configuration.md

```diff
@@ -42,27 +42,28 @@ Unless you need to set a non-default value, it is recommended to only populate o
-| `TIER_1_RATE_LIMIT`               | "100"         | Maximum restrictive request count limit used for expensive endpoints rate limiting.   |
-| `TIER_2_RATE_LIMIT`               | "800"         | Maximum moderate request count limit used for non expensive endpoints.   |
-| `TIER_3_RATE_LIMIT`               | "1600"        | Maximum relaxed request count limit used for static return endpoints.   |
+| Name                                  | Default       | Description                                                                                                                                                                                                                                                                                            |
+|---------------------------------------|---------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `CLIENT_TRANSPORT_SECURITY`           | "false"       | Flag to enable or disable TLS for both networks.                                                                                                                                                                                                                                                       |
+| `CONSENSUS_MAX_EXECUTION_TIME`        | "15000"       | Maximum time in ms the SDK will wait when submitting a transaction/query before throwing a TIMEOUT error.                                                                                                                                                                                              |
+| `DEFAULT_RATE_LIMIT`                  | "200"         | default fallback rate limit, if no other is configured.                                                                                                                                                                                                                                                |
+| `ETH_CALL_CACHE_TTL`                  | "200"         | Maximum time in ms to cache an eth_call response.                                                                                                                                                                                                                                                      |
+| `ETH_BLOCK_NUMBER_CACHE_TTL_MS`       | "1000"        | Time in ms to cache response from mirror node                                                                                                                                                                                                                                                          |
```

Comment:
I'm wondering out loud what the appropriate number should be for this cache expiration.  I believe our blocks are created at 2 second intervals which means at 1 second ( default value I understand ) we could be missing out on the latest block for a whole second?

Reply:
Yes, and that was why I agree that it must be configurable. Took the biggest alternative (in the [ticket](https://github.com/hashgraph/hedera-json-rpc-relay/issues/1057)) order to see if there are complaints, if so we can lower it to 500ms, but if not, 1000 would have the biggest (positive) impact on RPS limits.